src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -209,7 +209,7 @@ public void RebaseUriVisitor_VisitFileData_RebasesAllTheThings()
             visitor.FileDataKeys.Where(k => k != null && k.StartsWith(uriRootText)).Count().Should().Be(3);
             visitor.FileDataKeys.Where(k => k != null && k.StartsWith("#" + toolsRootBaseId + "#")).Count().Should().Be(1);
 
-            int uriCount = 16;
+            int uriCount = 17;
```

Comment:
>17 [](start = 27, length = 2)

Because we fixed that bug in the comprehensive sample that had the effect of adding another `fileLocation` object with a `uri`.

Reply:
I don't understand. The prerelease transformer should have created file locations from everything in the comprehensive sarif file. Are you masking a bug here because you brought the comprehensive sarif up to date in this regard?

---
In reply to: [244756655](https://github.com/Microsoft/sarif-sdk/pull/1198#discussion_r244756655) [](ancestors = 244756655)

Reply:
No, the additional file location is the one that you introduced in the 'mappedTo' property. the transformer is doing the right thing.

---
In reply to: [244871107](https://github.com/Microsoft/sarif-sdk/pull/1198#discussion_r244871107) [](ancestors = 244871107,244756655)

Reply:
Oh, I see.

---
In reply to: [244874264](https://github.com/Microsoft/sarif-sdk/pull/1198#discussion_r244874264) [](ancestors = 244874264,244871107,244756655)